### DISCUSSION TITLE
If I delete a project, will it free up the number for future ones?

### DISCUSSION THREAD

**Discussion by vizthex123** [community/community#124967] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Projects

### Body

For example, if I deleted project 1, then made a new project (either then or later) would it give the number 1 to the project i just made?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by gitstua** [comment, marked as answer]
Looks like the answer is no.

I just tested this and even though I deleted a project the next one got a new number.

**Reply by vizthex123** [reply]
_oof_
